src/AddressBook/ABPerson.cs

```diff
@@ -713,60 +713,84 @@ public void RemoveImage ()
 		[DllImport (Constants.AddressBookLibrary)]
 		extern static ABPersonCompositeNameFormat ABPersonGetCompositeNameFormat ();
 
-#if !NET
-		[Deprecated (PlatformName.iOS, 7, 0, message : "Use 'GetCompositeNameFormat (null)' instead.")]
-#else
+#if NET
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
 		[UnsupportedOSPlatform ("ios7.0")]
-#if IOS
+#if __MACCATALYST__
+		[Obsolete ("Starting with maccatalyst use the 'Contacts' API instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#elif IOS
 		[Obsolete ("Starting with ios7.0 use 'GetCompositeNameFormat (null)' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
 #endif
+#else
+		[Deprecated (PlatformName.iOS, 7, 0, message : "Use 'GetCompositeNameFormat (null)' instead.")]
 #endif
 		public static ABPersonCompositeNameFormat CompositeNameFormat {
 			get {return ABPersonGetCompositeNameFormat ();}
 		}
 
-#if !NET
-		[iOS (7,0)]
-#else
-		[UnsupportedOSPlatform ("ios7.0")]
-#if IOS
-		[Obsolete ("Starting with ios7.0 use 'GetCompositeNameFormat (null)' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#if NET
+		[SupportedOSPlatform ("ios7.0")]
+		[SupportedOSPlatform ("maccatalyst14.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
```

Comment:
Unsupported for maccatalyst but supported in maccatalyst 14.0 seems even more suspicous?